app/views/funding/what-funding.njk

```diff
@@ -25,30 +24,55 @@
 {% endblock %}
 
 {% block content %}
+  {% if model.error %}
+      {{ govukErrorSummary(model.error) }}
+    {% endif %}
+
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <form method="POST" action="/what-funding" novalidate>
         <input type="hidden" name="crumb" value="{{crumb}}"/>
         {% if model.items.length > 0 %}
-          {{ govukRadios(model) }}
-          {{ govukButton({
-          text: "Continue",
-          attributes: {
-            id: "submit"
-          }
-        }) }}
+          {{ govukCheckboxes(model) }}
```

Comment:
Mix of Nunjucks and HTML unfort here, could not find a way of mixing HTML elements with the Nunjucks component.

Reply:
I think it' unavoidable in this case